inductiva/molecules/scenarios/protein_solvation/post_processing.py

```diff
@@ -13,10 +13,11 @@
 
 FULL_TRAJECTORY_FILE = "full_trajectory.trr"
 COMPRESSED_TRAJECTORY_FILE = "compressed_trajectory.xtc"
-TOPOLOGY_FILE = "solvated_protein.tpr"
+PROTEIN_TOPOLOGY_FILE = "protein.gro"
+SYSTEM_TOPOLOGY_FILE = "solvated_protein.tpr"
 
 
-class ProteinSolvationOutput:
+class ProteinSolvationOutput():
```

Comment:
No need for this `()`. I'm actually not sure that this works as is.

Reply:
It works that I tested it but I will correct it :)